test/presenters/worldwide_organisation_presenter_test.rb

```diff
@@ -6,15 +6,25 @@ def schema_name
   end
 
   test "description of primary_role_person should have spaces between roles" do
-    presenter = create_presenter(WorldwideOrganisationPresenter, content_item: { "links" => { "primary_role_person" => [{ "details" => { "image" => {} }, "links" => { "role_appointments" => [{ "details" => { "current" => true }, "links" => { "role" => [{ "title" => "Example Role 1" }] } }, { "details" => { "current" => true }, "links" => { "role" => [{ "title" => "Example Role 2" }] } }] } }] } })
+    presenter = create_presenter(WorldwideOrganisationPresenter, content_item: { "links" => { "primary_role_person" => [{ "details" => { "image" => {} }, "links" => { "role_appointments" => [{ "details" => { "current" => true }, "links" => { "role" => [{ "content_id" => "1", "title" => "Example Role 1" }] } }, { "details" => { "current" => true }, "links" => { "role" => [{ "content_id" => "2", "title" => "Example Role 2" }] } }] } }], "roles" => [{ "content_id" => "1" }, { "content_id" => "2" }] } })
```

Comment:
This is an existing ailment of the test, but would there be value in adding a test to check that when a person doesn't have any current roles for this org the presenter doesn't blow up?

Reply:
If the persons role isn't in the filter list, they won't be in the content item at all, so this should be a non-problem. I'm wary of adding too many tests if they aren't needed.

I did wonder what happens if a role appointment doesn't have a role, but I don't think that is possible either, and that would have errored previously too